Bottom: 8a16658700cbe91c59070540126f57bac5049796
Top:    485d3177c7a08917bcc30e43e444b28fcf6bf752
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-16 00:55:53 +0000

Refresh of search-c-return-the-pad-who-s-

---

diff --git a/src/search.c b/src/search.c
index ebe94a6..5417462 100644
--- a/src/search.c
+++ b/src/search.c
@@ -193,7 +193,6 @@ pad_callback (const BoxType * b, void *cl)
     return 0;
 
   /* Determine how close our test-position was to the center of the pad  */
-  /* FIXME: This may not be a good enough heuristic */
   sq_dist = (PosX - (pad->Point1.X + (pad->Point2.X - pad->Point1.X) / 2)) *
             (PosX - (pad->Point1.X + (pad->Point2.X - pad->Point1.X) / 2)) +
             (PosY - (pad->Point1.Y + (pad->Point2.Y - pad->Point1.Y) / 2)) *
